undertow/src/main/java/org/wildfly/extension/undertow/ListenerResourceDefinition.java

```diff
@@ -154,8 +154,8 @@ abstract class ListenerResourceDefinition extends PersistentResourceDefinition {
     static final OptionAttributeDefinition RECEIVE_BUFFER = OptionAttributeDefinition.builder("receive-buffer", Options.RECEIVE_BUFFER).setAllowExpression(true).setValidator(new IntRangeValidator(1)).build();
     static final OptionAttributeDefinition SEND_BUFFER = OptionAttributeDefinition.builder("send-buffer", Options.SEND_BUFFER).setAllowExpression(true).setValidator(new IntRangeValidator(1)).build();
     static final OptionAttributeDefinition KEEP_ALIVE = OptionAttributeDefinition.builder("tcp-keep-alive", Options.KEEP_ALIVE).setAllowExpression(true).build();
-    static final OptionAttributeDefinition READ_TIMEOUT = OptionAttributeDefinition.builder("read-timeout", Options.READ_TIMEOUT).setDefaultValue(new ModelNode(60000)).setAllowExpression(true).setMeasurementUnit(MeasurementUnit.MILLISECONDS).build();
-    static final OptionAttributeDefinition WRITE_TIMEOUT = OptionAttributeDefinition.builder("write-timeout", Options.WRITE_TIMEOUT).setDefaultValue(new ModelNode(60000)).setAllowExpression(true).setMeasurementUnit(MeasurementUnit.MILLISECONDS).build();
+    static final OptionAttributeDefinition READ_TIMEOUT = OptionAttributeDefinition.builder("read-timeout", Options.READ_TIMEOUT).setDefaultValue(new ModelNode(1800000)).setAllowExpression(true).setMeasurementUnit(MeasurementUnit.MILLISECONDS).build();
+    static final OptionAttributeDefinition WRITE_TIMEOUT = OptionAttributeDefinition.builder("write-timeout", Options.WRITE_TIMEOUT).setDefaultValue(new ModelNode(1800000)).setAllowExpression(true).setMeasurementUnit(MeasurementUnit.MILLISECONDS).build();
```

Comment:
This probably requires a model transformation.

Reply:
I'm not very familiar with model transformation TBH. I had a look to WildFly undertow subsystem code base
and it seems to me the transformation is covered [here](https://github.com/wildfly/wildfly/blob/main/undertow/src/main/java/org/wildfly/extension/undertow/UndertowExtensionTransformerRegistration.java#L62) , isn't it @pferraro ? I also noticed I forgot to update wildfly-undertow_14_0.xsd with that new default value (previous push fixed that).

Reply:
Ah - you are correct.  I forgot that we only recently added default values for these attributes since WF28.